### DISCUSSION TITLE
[Feature] Save the search filter criteria

### DISCUSSION THREAD

**Discussion by amgschnappi** [immich-app/immich#8902] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I can search for person1 and person2. But this search cannot be saved. Please allow the save of this search criteria as a group. So next time, I just click this group1 and all the photos having both person1 and person2 are shown. Thanks.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
I think #1673 will cover this, right?

**Reply by amgschnappi** [reply]
I dont want to create any albums etc. I want to save the search criteria.

**Reply by amgschnappi** [reply]
I want to create multiple filters.
1. Kids
2. Kids with grandparents
3. Kids with grandparents_inlaws
4. Kids with wife
5. Kids with me
6. Me n wife 
7. Location1
8. Location2
9. Family location 3
etc
etc

I am not sure how album creation will solve my issue.
